docs/pages/desktop-access/reference/cli.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Desktop Access CLI Reference
+title: Windows-specific Command Reference
```

Comment:
Same here. A lot of these commands are for tctl, which we don't even ship for Windows, so saying Windows-specific seems wrong.